packages/widgets/Source/Viewer/Viewer.js

```diff
@@ -954,10 +887,6 @@ Either specify options.terrainProvider instead or set options.baseLayerPicker to
     pickAndSelectObject,
     ScreenSpaceEventType.LEFT_CLICK
   );
-  cesiumWidget.screenSpaceEventHandler.setInputAction(
-    pickAndTrackObject,
-    ScreenSpaceEventType.LEFT_DOUBLE_CLICK
-  );
```

Comment:
Given my other comment this should be brought back.

Reply:
Sure